packages/lib-classifier/src/store/AnnotationsStore.js

```diff
@@ -21,10 +21,18 @@ const AnnotationsStore = types
       self.reset()
     }
 
+    function createAnnotation(task) {
+      const taskAnnotation = task.createAnnotation()
+      self.annotations.put(taskAnnotation)
+      return taskAnnotation
+    }
+
     function addAnnotation (task, value) {
-      const annotation = self.annotation(task)
-      // new annotations must be added to this store before we can modify them
-      self.annotations.put(annotation)
+      let annotation = self.annotation(task)
```

Comment:
This line can be removed at some point, once we’re sure that `addAnnotation`is only called to add a new annotation. The TESS light curve viewer might still be using the old logic.